e2e/types/types.go

```diff
@@ -47,8 +47,8 @@ type Deployer interface {
 	GetName() string
 	// GetNamespace return the namespace for the ramen resources, or empty string if not using a special namespace.
 	GetNamespace(TestContext) string
-	// Return true for OCM discovered application, false for OCM managed applications.
-	IsDiscovered() bool
```

Comment:
Why? we have exactly one disapp deployer, and it can return true in all cases. Why do we need to add a helper function when the deployer can tell that?